examples/one-rate-plan/version.tf

```diff
@@ -6,7 +6,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "ibm-cloud/ibm"
-      version = "1.56.1"
+      version = "1.60.0"
```

Comment:
please change this back. As per code comment above, this example should lock into the lowest version that provider supports, which in this case is 1.56.1